src/date-navigator/date-navigator.style.tsx

```diff
@@ -1,36 +1,56 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import { Color } from "../color";
+import { MediaQuery } from "../media";
 import { ClickableIcon } from "../shared/clickable-icon";
 import { Text } from "../text";
 
 export const HeaderArrowButton = styled(ClickableIcon)`
     height: 2.5rem;
-    padding: 1rem;
+    width: 2.5rem;
+    padding: 0;
+    margin: 0.5rem 0.5rem;
+    border-radius: 0.25rem;
+    outline: 1px solid ${Color.Neutral[5]};
+    background-color: ${Color.Neutral[8]};
+    :hover {
+        background-color: ${Color.Neutral[7]};
+    };
     :disabled {
         cursor: not-allowed;
-        > svg {
-            color: ${Color.Neutral[5]};
-        }
-    }
+        background-color: ${Color.Neutral[6]};
+    };
 `;
 
 export const Container = styled.div`
     display: flex;
+    position: relative;
     align-items: center;
-    justify-content: center;
-    width: 100%;
-`;
-
-export const Wrapper = styled.div`
-    text-align: center;
+    justify-content: space-between;
+    background-color: ${Color.Neutral[7]};
+    min-width: 320px;
+    height: 56px;
+    width: 368px;
+    border-radius: 0.25rem;
+    ${MediaQuery.MaxWidth.tablet} {
+        width: 100vw;
+    }
 `;
 
-export const StyledDateText = styled(Text.H4)`
-    color: ${Color.Neutral[1]};
+export const StyledDateText = styled(Text.H4) <{ $enableDateClick?: boolean }>`
```

Comment:
this should probably be a button, so that keyboard users can tab to it and toggle the calendar